presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/SymbolMapper.java

```diff
@@ -228,6 +237,20 @@ public PlanNode visitApply(ApplyNode node, RewriteContext<Void> context)
                     mapAndDistinct(node.getCorrelation()));
         }
 
+        @Override
+        public PlanNode visitValues(ValuesNode node, RewriteContext<Void> context)
+        {
+            ValuesNode intermediate = new ValuesNode(
+                    planNodeIdAllocationStrategy.getId(node),
+                    node.getOutputSymbols(),
+                    node.getRows().stream()
```

Comment:
Is it necessary to map the rows?  I assumed they were always literals, but I guess the types don't ensure that.